client/my-sites/checkout/composite-checkout/record-analytics.js

```diff
@@ -217,7 +217,7 @@ export default function createAnalyticsEventHandler( reduxDispatch ) {
 					recordTracksEvent( 'calypso_checkout_composite_add_coupon_clicked', {} )
 				);
 
-			case 'STEP_NUMBER_CHANGED':
+			case 'STEP_NUMBER_CHANGED': {
```

Comment:
It's not a problem, but what's the goal of adding curly braces around this case? The main benefit I'm aware of is to create a variable scope so you can define new variables, but there are none in this block. If it's just for visual organization, it seems a little unnecessary to me since `case` blocks are already indented, but maybe it's for future-proofing, which I can see the argument for.